scene/resources/physics_material.cpp

```diff
@@ -43,13 +43,14 @@ void PhysicsMaterial::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_absorbent", "absorbent"), &PhysicsMaterial::set_absorbent);
 	ClassDB::bind_method(D_METHOD("is_absorbent"), &PhysicsMaterial::is_absorbent);
 
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "friction", PROPERTY_HINT_RANGE, "0,1,0.01,or_greater"), "set_friction", "get_friction");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "friction", PROPERTY_HINT_RANGE, "0,1,0.001"), "set_friction", "get_friction");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "rough"), "set_rough", "is_rough");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "bounce", PROPERTY_HINT_RANGE, "0,1,0.01,or_greater"), "set_bounce", "get_bounce");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "bounce", PROPERTY_HINT_RANGE, "0,1,0.001"), "set_bounce", "get_bounce");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "absorbent"), "set_absorbent", "is_absorbent");
 }
 
 void PhysicsMaterial::set_friction(real_t p_val) {
+	ERR_FAIL_COND(p_val > 1.0);
```

Comment:
Add a human-readable error message:

```suggestion
	ERR_FAIL_COND_MSG(p_val > 1.0, "Can't set PhysicsMaterial friction above 1.0.");
```

Reply:
@Calinou I also changed the bounce because the documentation says that the value must be at most 1.0. Thinking about it, the bounce can actually be greater than 1.0. Only the friction must be limited to 1.0. It has to be changed in the documentation. This PR can be merged then changing only the FRICTION and tomorrow, I can change the doc in the XML for you regarding the bounce then. Is it possible?

Reply:
I think a bounce of > 1 makes no sense as it increases the system energy and is not possible in the real world (unless the bouncing object has some internal mechanical construct that propels it).

A friction of > 1 is however possible in the real world. For normal smooth surface objects a friction limit of 1 makes sense, but if you cover both objects with e.g. velcro then the friction can go way above 1 (the linear friction model would not be accurate in that case though).

Reply:
For what it is worth, I'm working on a physics based game and being able to increase the friction past 1 has massively improved the gameplay. If the friction is to be capped, I think it should at least be higher than 1. I use a value of 3 for accurately recreating bicycle tire grip.